docs/deployments/networking.md

```diff
@@ -0,0 +1,11 @@
+# Networking
+
+_WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
+
+## API Gateway
+
+APIs are deployed with an internet-facing API Gateway by default (the API Gateway forwards requests to the API load balancer). Each API can be independently configured to not create the API Gateway endpoint by setting `api_gateway: none` in the `networking` field of the [api configuration](api-configuration.md). If the API Gateway endpoint is not created, your API can still be accessed via the API load balancer; `cortex get API_NAME` will show the load balancer endpoint if API Gateway is disabled.
+
+If you'd like to force traffic to go through your public API Gateway endpoint, set `api_load_balancer_scheme: internal` in your [cluster configuration](../cluster-management/config.md) file before creating your cluster.
+
+If you'd like to ensure that an API is not publicly accessible, set `api_load_balancer_scheme: internal` in your [cluster configuration](../cluster-management/config.md) file, and set `api_gateway: none` in the `networking` field of your [api configuration](api-configuration.md). If you do this, you will need to configure [VPC Peering](../guides/vpc-peering.md) to make prediction requests to your APIs.
```

Comment:
maybe a table?
                      HTTPS ENDPOINT                                  | HTTP ENDPOINT
____________|______________________________________________________ ___ 
API Public   | api_load_balancer_scheme: internal`   | api_load_balancer_scheme: internet facing`
                    | api_gateway: public                               |   api_gateway: none
____________ |______________________________________________________________
API Private  | ------------------------------------   | api_load_balancer_scheme: internal`
                    | ------------------------------- ----- |   api_gateway: none

Reply:
Good idea, I just added it!